src/main/java/seedu/address/logic/commands/calendars/ShowCommand.java

```diff
@@ -22,24 +21,22 @@
 
     public static final String MESSAGE_SUCCESS = "Showed calendar";
     public static final String MESSAGE_USAGE = getCommandFormat(COMMAND_WORD)
-            + " <year> <month>: Shows the calendar view. "
-            + "Parameters: " + PREFIX_YEAR + "YEAR " + PREFIX_MONTH + "MONTH";
+            + " <year> <month>: Shows the calendar view. " + "Parameters: " + PREFIX_YEAR + "YEAR " + PREFIX_MONTH
```

Comment:
I think it's better for you to follow the MESSAGE_USAGE of other `extends Command` classes, which are usually:
`": <purpose of this>.\n" + "Parameters: <params>\n" + "Example: "`